sdk/nodejs/queryable/index.ts

```diff
@@ -0,0 +1,57 @@
+// Copyright 2016-2018, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { OutputInstance } from "../index";
+import { Resource } from "../resource";
+
+/**
+ * ResovledResource is a `Resource` with all fields containing `Output` values fully resolved. This
+ * is useful primarily when we're querying over resource outputs (e.g., using
+ * `pulumi.runtime.listResourceOutputs`), and we expect all values to be present and fully-resovled.
+ */
+export type ResovledResource<T extends Resource> = Omit<Resolved<T>, "urn" | "getProvider">;
```

Comment:
Should this also be `Reso_lv_edResource<T extends...` instead of `Reso_vl_edResource<T ....`?